common/djangoapps/track/backends/tests/test_oli.py

```diff
@@ -0,0 +1,201 @@
+                                                                                                          'answervariable': None,
+                                                                                                          'msg': '',
+                                                                                                          'queuestate': None}},
+                           'state': {'student_answers': {u'i4x-edX-DemoX-problem-c554538a57664fac80783b99d9d6da7c_2_1': u'[231,198]'},
+                                     'seed': 1,
+                                     'done': True,
+                                     'correct_map': {u'i4x-edX-DemoX-problem-c554538a57664fac80783b99d9d6da7c_2_1': {'hint': u'',
+                                                                                                                     'hintmode': None,
+                                                                                                                     'correctness': u'incorrect',
+                                                                                                                     'npoints': None,
+                                                                                                                     'answervariable': None,
+                                                                                                                     'msg': u'',
+                                                                                                                     'queuestate': None}},
+                                     'input_state': {u'i4x-edX-DemoX-problem-c554538a57664fac80783b99d9d6da7c_2_1': {}}},
+                           'answers': {u'i4x-edX-DemoX-problem-c554538a57664fac80783b99d9d6da7c_2_1': u'[480,237]'},
+                           'attempts': 71,
+                           'max_grade': 1,
+                           'problem_id': u'i4x://edX/DemoX/problem/c554538a57664fac80783b99d9d6da7c'},
+                 'event_source': 'server',
+                 'context': {'course_user_tags': {},
+                             'user_id': self.user.pk,
+                             'org_id': 'edX',
+                             'module': {'usage_key': u'i4x://edX/DemoX/problem/c554538a57664fac80783b99d9d6da7c',
+                                        'display_name': u'Pointing on a Picture'},
+                             'course_id': u'edX/DemoX/Demo_Course2',
+                             'path': u'/courses/edX/DemoX/Demo_Course/xblock/i4x:;_;_edX;_DemoX;_problem;_c554538a57664fac80783b99d9d6da7c/handler/xmodule_handler/problem_check'},
+                 'time': datetime.datetime.now,
+                 'page': 'x_module'}
+
+        message = self.backend.send(event)
+        # this should not be processed and message should be None.
+        self.assertEqual(message, None)
+
+
+class MockResponse(object):
```

Comment:
you should be able to use Mock instead of having to define a stub here. `Mock(content = "string")`